### ISSUE TITLE
1.0.189-master  misc bundle crashes resolve

### ISSUE THREAD

**Issue by manuliner** [NatronGitHub/openfx-misc#74]
downloaded last appvejor build and tested with resolve 16.

probably  ->>>    propGetDimension: unknown object (NatronOfxPropNativeOverlays)

[davinci_resolve.log](https://github.com/NatronGitHub/openfx-misc/files/4592302/davinci_resolve.log)

**Comment by devernay** [comment]
that's not this one, sorry (the plugin is just asking if the property exists).
The plugins load load fine on macOS Resolve, so I can't debug this.
Is there a crash report of some sort?

**Comment by manuliner** [comment]
i added the davinci log in my first post. Does this help?

**Comment by devernay** [comment]
nope, the log doesn't help either. Resolve has very poor OFX support.
It works fine on Resolve 16 on macOS, so I cannot debug here
